client/src/locales/en-US.json

```
@@ -1567,18 +1567,19 @@
             "admin": "Administrators | Administrator | Administrators",
             "standard": "Members | Member | Members",
             "outsider": "Externals | External | Externals",
-            "consumptionDetail": "Consumption detail",
-            "nonPaying": "Non paying",
-            "paying": "charged",
-            "free": "free",
-            "ofWhich": "of which",
-            "data": "data",
-            "metadata": "metadata",
+            "total" : "Total",
+            "detail": "Detail",
+            "consumption": "Consumption",
+            "includedTitle": "Included",
+            "additionalTitle": "Additional",
```

IMO "Additional" seems like it is also included (like "your plan has 10 GB + 1 additional GB"). If that's the case, maybe "Extra" is a better term.
```suggestion
            "additionalTitle": "Extra",
```

On the contrary, if it means something that is "exceeding" the authorized limit, maybe we should use "Exceeding"
```suggestion
            "additionalTitle": "Exceeding",
```